src/app/service/squad/LaunchSquad.ts

```diff
@@ -66,15 +66,12 @@ const getTitle = async (member: GuildMember, ctx?: CommandContext): Promise<void
 	collector.on('end', async (_, reason) => {
 
 		// if getTitlePrompt is not the last message, time out silently.
-		if ((getTitlePrompt.id === dmChannel.lastMessage.id) && (reason === 'time')) {
+		if ((getTitlePrompt.id === dmChannel.lastMessageId) && (reason === 'time')) {
 			Log.debug('squadUp getTitle message collector timed out');
 
-			try {
-				await dmChannel.send('The conversation timed out.');
-			} catch (e) {
-				Log.debug(`Squad getTitle collector timed out, unable to send dm, error msg: ${e}`);
-			}
+			await dmChannel.send('The conversation timed out.');
```

Comment:
This looks much cleaner, @tikisailor, thank you. However, we do want all `async/await` wrapped in a `try/catch`. The idea behind my last review wasn't to remove most `try/catch` statements, but rather work the code over so we don't have _nested_ `try/catch` statements. Correct me if I'm wrong, @amaredeus: if we have an async function with nested awaits, we should be able to "catch" all errors simply by wrapping the top level function in a `try/catch`. If it comes to a matter of logging specifics, it may make sense to refactor accordingly.